docs/src/pages/guides/testing/testing.md

```diff
@@ -2,166 +2,20 @@
-
 ## Userspace
 
-What about writing tests in userspace? The Material-UI styling infrastructure uses some helper functions built on top of [enzyme](https://github.com/airbnb/enzyme) to make the process easier, which we are exposing. You can take advantage of them if you so choose.
-We use almost exclusively full DOM rendering APIs. We encourage you to do the same especially
-if your components rely on custom themes. Tests using shallow rendering APIs become more brittle
-with the amount of provider components they require.
-
-### Full DOM rendering
-
-Full DOM rendering is ideal for use cases where you have components that may interact with DOM APIs or may require the full lifecycle in order to fully test the component (e.g., `componentDidMount` etc.).
-
-The `createMount()` function is provided for this situation.
-Aside from wrapping the enzyme API, it provides a `cleanUp` function.
-
-### Shallow rendering
-
-Shallow rendering is useful to constrain your testing to a component as a unit. This also ensures that your tests aren't indirectly asserting behavior of child components.
-Shallow rendering was created to test components in isolation. This means without leaking child implementation details such as the context.
-
-The `createShallow()` function can be used for this situation. Aside from wrapping the enzyme API, it provides a `dive` and `untilSelector` option.
-
-### Render to string
-
-Rendering to a string is useful to test the behavior of the components that are used on the server.
-You can take advantage of this to assert the generated HTML string.
-
-The `createRender()` function is ideal for this. This is just an alias for the enzyme API, which is only exposed for consistency.
-
-## API
-
-### `createMount([options]) => mount`
-
-Generate an enhanced mount function with the needed context.
-Please refer to the [enzyme API documentation](https://airbnb.io/enzyme/docs/api/mount.html) for further details on the `mount` function.
-
-#### Arguments
-
-1. `options` (_Object_ [optional])
-
-- `options.mount` (_Function_ [optional]): The mount function to enhance, it uses **enzyme by default**.
-- The other keys are forwarded to the options argument of `enzyme.mount()`.
-
-#### Returns
-
-`mount` (_mount_): A mount function.
-
-#### Examples
-
-```jsx
-import { createMount } from '@material-ui/core/test-utils';
-import { ThemeProvider } from '@material-ui/core/styles';
-
-describe('<MyComponent />', () => {
-  let mount;
-
-  function MySuccessButton({ children }) {
-    return (
-      <ThemeProvider
-        theme={{
-          success: { main: '#fff' },
-        }}
-      >
-        {children}
-      </ThemeProvider>
-    );
-  }
+We recommend testing your application without tying the tests too close to Material-UI.
+This is how we test our components internally.
+A library that has a first-class API for this approach is [`@testing-library/react`](https://testing-library.com/docs/react-testing-library/intro).
```

Comment:
I wonder if this sentence should come at the end of the section, after the approach has been fully explained? The prior sentences only allude to what *not* to do...

Reply:
That makes more sense :+1: I moved it after the example.